internal/fxlog/fxlog.go

```diff
@@ -58,6 +58,14 @@ func (l *Logger) PrintProvide(t interface{}) {
 	}
 }
 
+// PrintSupply logs a type supplied directly into the dig.Container
+// by the given constructor function.
+func (l *Logger) PrintSupply(constructor interface{}) {
```

Comment:
Not a problem in this PR but this exposed that we weren't logging names/groups for fx.Annotated constructors at all. Created #703 for us to fix that.